src/partisan_hyparview_peer_service_manager.erl

```diff
@@ -462,7 +478,9 @@ handle_info({'EXIT', From, _Reason},
                           connections=Connections}};
 
 handle_info({connected, Peer, Tag, _RemoteState}, State) ->
-    handle_connect(Peer, Tag, State);
+    %% @todo retreive the PeerEpoch from the Peer.
```

Comment:
In order the change the message to contain the `Epoch`, the `get_local_state()` needs to return the epoch value as well. Currently it only returns a set of active view. Can I change the `get_local_state()` function? It looks like that function is an interface, so it can affect other managers as well.